### DISCUSSION TITLE
CSV parser

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#940] [Support / Q&A] [answered]
While trying to solve the issue: https://github.com/rowyio/rowy/discussions/746
I am getting a dependencies error when installing iconv-lite package. @notsidney 

`npm WARN config global --global, --local are deprecated. Use --location=global instead.
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @craco/craco@6.4.3
npm ERR! Found: react-scripts@5.0.0
npm ERR! node_modules/react-scripts
npm ERR!   peer react-scripts@"^5" from craco-swc@0.5.1
npm ERR!   node_modules/craco-swc
npm ERR!     dev craco-swc@"^0.5.1" from the root project
npm ERR!   react-scripts@"^5.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react-scripts@"^4.0.0" from @craco/craco@6.4.3
npm ERR! node_modules/@craco/craco
npm ERR!   peer @craco/craco@"^6  ^7" from craco-swc@0.5.1
npm ERR!   node_modules/craco-swc
npm ERR!     dev craco-swc@"^0.5.1" from the root project
npm ERR!   dev @craco/craco@"^6.4.3" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react-scripts@4.0.3
npm ERR! node_modules/react-scripts
npm ERR!   peer react-scripts@"^4.0.0" from @craco/craco@6.4.3
npm ERR!   node_modules/@craco/craco
npm ERR!     peer @craco/craco@"^6  ^7" from craco-swc@0.5.1
npm ERR!     node_modules/craco-swc
npm ERR!       dev craco-swc@"^0.5.1" from the root project
npm ERR!     dev @craco/craco@"^6.4.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\RajGM\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RajGM\AppData\Local\npm-cache_logs\2022-09-18T17_43_50_307Z-debug-0.log`

**Comment by notsidney** [comment, marked as answer]
Please add new dependencies using Yarn.
